src/Sulu/Bundle/AdminBundle/Resources/js/components/Radio/Radio.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+import * as React from 'react';
```

Comment:
Please use `import React from 'react'` and import the types using `import type {Type1, Type2} from 'react'`.